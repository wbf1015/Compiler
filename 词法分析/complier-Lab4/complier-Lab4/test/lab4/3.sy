int main(){
    int a,b,c;
    a=0;
    b=1;
    c=2;

    a>=3;
    c<=a;
    a!=1;

    a|b;
    a&2;
    a%1;

    a||4;
    a&&0;
}