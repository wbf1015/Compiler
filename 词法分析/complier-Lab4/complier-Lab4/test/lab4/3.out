[DEBUG LAB4]: 	INT		int	1	0
[DEBUG LAB4]: 	ID		main	1	4	94900446061440
[DEBUG LAB4]: 	LPAREN		(	1	8
[DEBUG LAB4]: 	RPAREN		)	1	9
[DEBUG LAB4]: 	LBRACE		{	1	10
[DEBUG LAB4]: 	INT		int	2	4
[DEBUG LAB4]: 	ID		a	2	8	94900446061760
,[DEBUG LAB4]: 	ID		b	2	9	94900446061920
,[DEBUG LAB4]: 	ID		c	2	10	94900446062032
[DEBUG LAB4]: 	SEMICOLON		;	2	11
[DEBUG LAB4]: 	ID		a	3	4	94900446061760
[DEBUG LAB4]: 	ASSIGN		=	3	5
[DEBUG LAB4]: 	DECIMIAL_INT		0	3	6	0.000000
[DEBUG LAB4]: 	SEMICOLON		;	3	7
[DEBUG LAB4]: 	ID		b	4	4	94900446061920
[DEBUG LAB4]: 	ASSIGN		=	4	5
[DEBUG LAB4]: 	DECIMIAL_INT		1	4	6	1.000000
[DEBUG LAB4]: 	SEMICOLON		;	4	7
[DEBUG LAB4]: 	ID		c	5	4	94900446062032
[DEBUG LAB4]: 	ASSIGN		=	5	5
[DEBUG LAB4]: 	DECIMIAL_INT		2	5	6	2.000000
[DEBUG LAB4]: 	SEMICOLON		;	5	7
[DEBUG LAB4]: 	ID		a	7	4	94900446061760
[DEBUG LAB4]: 	MOREOREQUALE		>=	7	5
[DEBUG LAB4]: 	ID		b	7	7	94900446061920
[DEBUG LAB4]: 	SEMICOLON		;	7	8
[DEBUG LAB4]: 	ID		c	8	4	94900446062032
[DEBUG LAB4]: 	LESSOREQUAL		<=	8	5
[DEBUG LAB4]: 	ID		a	8	7	94900446061760
[DEBUG LAB4]: 	SEMICOLON		;	8	8
[DEBUG LAB4]: 	ID		a	9	4	94900446061760
[DEBUG LAB4]: 	UNEQUAL		!=	9	5
[DEBUG LAB4]: 	ID		b	9	7	94900446061920
[DEBUG LAB4]: 	SEMICOLON		;	9	8
[DEBUG LAB4]: 	ID		a	11	4	94900446061760
[DEBUG LAB4]: 	LOGIC_OR		|	11	5
[DEBUG LAB4]: 	ID		b	11	6	94900446061920
[DEBUG LAB4]: 	SEMICOLON		;	11	7
[DEBUG LAB4]: 	ID		a	12	4	94900446061760
[DEBUG LAB4]: 	LOGIC_AND		&	12	5
[DEBUG LAB4]: 	ID		b	12	6	94900446061920
[DEBUG LAB4]: 	SEMICOLON		;	12	7
[DEBUG LAB4]: 	ID		a	13	4	94900446061760
[DEBUG LAB4]: 	MOD		%	13	5
[DEBUG LAB4]: 	ID		b	13	6	94900446061920
[DEBUG LAB4]: 	SEMICOLON		;	13	7
[DEBUG LAB4]: 	ID		a	15	4	94900446061760
[DEBUG LAB4]: 	OR		||	15	5
[DEBUG LAB4]: 	ID		b	15	7	94900446061920
[DEBUG LAB4]: 	SEMICOLON		;	15	8
[DEBUG LAB4]: 	ID		a	16	4	94900446061760
[DEBUG LAB4]: 	AND		&&	16	5
[DEBUG LAB4]: 	ID		b	16	7	94900446061920
[DEBUG LAB4]: 	SEMICOLON		;	16	8
[DEBUG LAB4]: 	RBRACE		}	17	0
